src/main/java/appeng/integration/modules/jei/GrinderRecipeWrapper.java

```diff
@@ -50,14 +50,8 @@ public void getIngredients( IIngredients ingredients )
 		ingredients.setInput( ItemStack.class, recipe.getInput() );
 		List<ItemStack> outputs = new ArrayList<>( 3 );
 		outputs.add( recipe.getOutput() );
-		if( recipe.getOptionalOutput() != null )
-		{
-			outputs.add( recipe.getOptionalOutput() );
-		}
-		if( recipe.getSecondOptionalOutput() != null )
-		{
-			outputs.add( recipe.getSecondOptionalOutput() );
-		}
+		recipe.getOptionalOutput().ifPresent( itemStack -> outputs.add( itemStack ) );
+		recipe.getSecondOptionalOutput().ifPresent( itemStack -> outputs.add( itemStack ) );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1612)